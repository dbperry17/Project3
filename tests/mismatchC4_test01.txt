TYPE
   q, r, s: BOOLEAN;
   u, v : LONG;
VAR
   a : r;
   b : q;
   c : u;
   d : v;
{
    WHILE d {
         c = c + d;
    }
}

