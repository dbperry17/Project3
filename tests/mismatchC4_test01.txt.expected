TYPE MISMATCH 10 C4
